dhis-2/dhis-web-api/src/main/resources/openapi/ProgramMessageController.md

```diff
@@ -0,0 +1,61 @@
+# Send Message
+
+## Specific endpoints
+
+### `sendMessages`
+
+Send event or enrollment messages.
+
+### `getScheduledSentMessage`
+
+Get all of those scheduled messages which were sent successfully.
+
+### `getProgramMessages`
+
+Get program messages matching given query criteria.
+
+## List of supported parameters
+
+### `*.parameter.ou`
+
+Get program messages for given set of OrganisationUnits
+
+### `*.parameter.enrollment`
+
+Get program messages for given enrollment
+
+### `*.parameter.event`
+
+Get program messages for given event
+
+### `*.parameter.programInstace`
+
+Get program messages for given enrollment
+
+### `*.parameter.programStageInstace`
+
+Get program messages for given event
+
+### `*.parameter.afterDate`
+
+Get program messages after given date
+
+### `*.parameter.beforeDate`
+
+Get program messages before given date
+
+### `*.parameter.messageStatus`
+
+Get program messages based on status for example SENT, SCHEDULED etc
+
+### `*.parameter.page`
+
+Number of pages to be loaded
+
+### `*.parameter.pageSize`
+
+Number of records per page
+
+### `*.parameter.ProgramMessageBatch.programMessages`
```

Comment:
Is this a parameter?
Most of our examples are GET endpoints, I am not sure how to represent the `body`.
@jbee How should this be specified?

Reply:
It is just `{endpoint}.request.description` for the request body

Reply:
@zubaira Can you change it accordingly? And move it under the specific endpoint description?